docs/apis-tools/camunda-api-rest/camunda-api-rest-guidelines.md

```diff
@@ -0,0 +1,309 @@
+
+Search endpoints can also be used to directly access entity instances with a unique identifier. As an alternative, resources can also provide GET method endpoints for fetching the data of single instances. In most cases, this is done by a specific key parameter in the URL path, e.g. _GET /v2/users/{userKey}_.
+
+### Search requests
+
+Query requests consist of the components for **filter**, **sort**, and **page**.
+
+The **filter** object defines which fields should match. Only items that match the given fields will be returned. The available fields vary by object and are described in the respective search endpoint. Filtering by a unique identifier is usually available in filtering options. Beyond that, the filter options don’t have to comprise all the returned items’ attributes.
+
+The **sort** object specifies which fields of the object should be sorted and whether they are sorted in ascending (ASC) or descending (DESC) order.
+
+The **page** object details how to slice the result set. An initial search request can omit the page object or define the _limit_. This specifies the maximum number of results to retrieve per request. Subsequent requests can use the value of the returned _firstItemSortValues_ and _lastItemSortValues_ of the [search responses](#search-responses) to page through the items by copying that array into one of the attributes _searchAfter_ or _searchBefore_.
+
+```
+{
+  "filter":  {
+    <entity-specific  filter  options>
+  },
+  "sort":  [
+    {  "<field>":  "<order>"  },
+    {  "<field>":  "<order>"  }
+  ],
+  "page":  {
+    "searchAfter":  [  …  ,  …  ],
+    "searchBefore":  [  …  ,  …  ],
+    "limit":  <limit>
+  }
+}
+```
+
+#### Advanced Search
+
+To provide an easy yet expressive way for users to search for and filter resources, search requests can contain more advanced filter criteria than fields being _equal_ to a target value.
+
+For example, this allows searching using logical (and, in) and comparison operators (greater than, less than). The list of supported advanced filter operators is described below. Examples of advanced operators are shown in the [Search example](#search-example).
+
+##### Conditional operators
+
+| Operator | Syntax                                    | Description                                                                                                                                               |
+| -------- | ----------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| $eq      | field: { "$eq": value }                   | Filter where field is equal to value. Abbreviated form field: value is also allowed.                                                                      |
```

Comment:
```suggestion
| `$eq`      | `field: { "$eq": value }`                   | Filter where field is equal to value. Abbreviated form field: value is also allowed.                                                                      |
```

Potentially blocking:

Again, this is my interpretation of the style guide, but I think these bits should all be inline code. (Applies also to many lines that follow this one.)